pages/docs/installation/installation-special-systems.md

```diff
@@ -794,3 +794,35 @@ export PRECICE_BOOST_ROOT=$HOME/software/boost_1_53_0
 * configure ALYA with `-L[PathToPreCICE]/build/last -lprecice -lstdc++ -lrt`
 * for running: also put `module load intel` in your jobscript
 * use `network="ib0"` for sockets communication beyond one node
+
+### Max Planck Computing and Data Facility (MPCDF) Cobra cluster
+
+#### Installing dependencies
+
+##### Eigen3
+
+See the [Eigen dependency section](https://precice.org/installation-source-dependencies.html#eigen).
+
+#### Installing preCICE
+
+On the Cobra cluster preCICE is installed from source. Please set the installation prefix paths as shown [in this section](https://precice.org/installation-source-preparation.html#installation-prefix) before installing preCICE. The preCICE repository is either cloned or copied to the cluster, and then the following commands are run:
+
+```bash
+module purge
+module load gcc/9 impi/2019.7 cmake anaconda boost/1.74
+module list
+
+rm -rf build
+mkdir -p build && cd build
+cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$PRECICE_ROOT -DPRECICE_PETScMapping=OFF -DPRECICE_PythonActions=OFF \
+  -DMPI_CXX_COMPILER=mpigcc -DCMAKE_BUILD_TYPE=Debug ..
+make -j20
+
+LIB=$PRECICE_ROOT/libprecice.so
```

Comment:
`PRECICE_ROOT` is undefined at this point.

Using `PRECICE_ROOT` was heavily used in the past (internally needed for tests only). It also always pointed to the root of the repository, not at the install directory. So this could lead to confusion with more seasoned preCICE users.